tests/basic/sharedv4_svc_test.go

```diff
@@ -420,6 +441,16 @@ var _ = Describe("{Sharedv4SvcFunctional}", func() {
 			})
 			testFailoverFailback()
 		})
+
+		// test failover/failback by recover the node
+		Context("{Shared4SvcRecoverNode}", func() {
+			BeforeEach(func() {
+				namespacePrefix = "recovernode-"
```

Comment:
recovernode -> nodemaintenance 

Also, we don't need - in the end any more.

Reply:
updated